src/fileiter.cpp

```diff
@@ -53,6 +53,12 @@ namespace std{
 #include <sys/cygwin.h>
 #endif
 
+#if defined (BOOST_ASIO_WINDOWS_RUNTIME) || defined (BOOST_PLAT_WINDOWS_PHONE)
```

Comment:
> I'd like to make the change to the #ifdef statement. I'm relatively 
> new to Git: how can I make that change? Do I need to submit a new pull 
> request?
>

No just push the change to the cloned repro you used for the PR and the 
PR will get updated automatically.

Thanks, John.